docs/getting_started/advanced_setup_guides.rst

```diff
@@ -76,6 +76,29 @@ For more details about fastapi and uvicorn, see `here <https://fastapi.tiangolo.
 
 Fastapi also provides beautiful REST API docs that you can check at `http://localhost:6900/api/docs <http://localhost:6900/api/docs>`__.
 
+Environment variables
+---------------------
+
+Here we will give you a list of useful environment variables that you can set to configure your client or server.
+
+Client
+^^^^^^
+
+- ``RUBRIX_API_URL``: The default API URL when calling :meth:`rubrix.init`.
+
+- ``RUBRIX_API_KEY``: The default API key when calling :meth:`rubrix.init``.
+
+- ``RUBRIX_WORKSPACE``: The default workspace when calling :meth:`rubrix.init``.
```

Comment:
we should include at least:

- ELASTICSEARCH : The elasticsearch connection endpoint (Default: http://localhost:9200)
- RUBRIX_ELASTICSEARCH_SSL_VERIFY: Check ssh connection to elasticsearch backend. Default, True. You can disable it to disable ssl certificate verification
- RUBRIX_NAMESPACE: A namespace prefix used for manage elasticsearch indices. You can use this namespace to use the same elasticsearch iinstance for several independent rubrix instances.
- RUBRIX_DEFAULT_ES_SEARCH_ANALYZER: Default analyzer for textual fields (non metadata) 
- RUBRIX_EXACT_ES_SEARCH_ANALYZER: Default analyzer for `*.exact` fields in textual information.

Reply:
Done